spec/support/contexts/response_form_conditional_logic.rb

```diff
@@ -0,0 +1,108 @@
+# frozen_string_literal: true
+
+shared_context "response form conditional logic" do
```

Comment:
Nice job moving this into a shared file. You don't actually need the module part if you're doing `shared_context`. Only if you have a file of helpers methods that you want included in _all_ specs of a certain type.

I recently reorg'd the spec/support folder to make this clearer...